substrate/client/network/sync/src/service/mod.rs

```diff
@@ -18,6 +18,6 @@
 
 //! `ChainSync`-related service code
 
-pub mod chain_sync;
+pub mod engine;
```

Comment:
Why is `SyncingEngine` moved to service? I don't think this makes sense. I imagined `service` to be for files that pertain to services provide by syncing, such as `NetworkServiceProvider` and `SyncingService`. If there's now tighter coupling between `SyncingService` and `SyncingEngine` and you really want to move files, I think it'd make more sense to have `engine` directory but even that's a push IMO. I thought it was fine as it was before.

Reply:
It's just the `SyncingService` in the `service` directory. `SyncingEngine` itself is left in the directory root.